test/config/type/function-with-env/a.js

```diff
@@ -1 +1,3 @@
 console.log('chuntaro');
+// eslint-disable-next-line no-undef
```

Comment:
need disable because envMessage is not present in scope but instead coming from env